lib/lita/handlers/jls.rb

```diff
@@ -137,6 +143,72 @@ def cla(msg)
         push_exception(e, :project => "#{user}/#{project}", :pr => pr)
       end
 
+      def migrate_pr(msg)
+        source_url = msg.match_data["source_url"]
+        destination_url = msg.match_data["destination_url"]
+        if source_url.nil? || destination_url.nil?
+          raise "Invalid paramaters provided #{msg}"
+        end
+
+        pr_num = source_url.strip.split("/").last
+        if pr_num.nil? || pr_num !~ /\d+/
+          raise "Invalid pull request provided #{source_url}"
+        end
+
+        destination_github_parser = parse_github_url(destination_url)
+        source_github_parser = parse_github_url(source_url)
+
+        source_github_pr = github_get_pr("#{source_github_parser.user}/#{source_github_parser.project}", pr_num)
+
+        # Clone destination dir, patch and then push branch
+        repository = LitaJLS::Repository.new(destination_github_parser)
+        repository.clone if Dir["#{repository.git_path}/*"].empty?
+        repository.switch_branch("master")
+
+        # create a branch like pr/1234
+        pr_branch = "pr/#{pr_num}"
```

Comment:
Can we add a word to identify that the branch was created by the bot?
I would suggest `migrated-pr/1234` or something better.?